tests/sentry/sentry_apps/tasks/test_sentry_apps.py

```diff
@@ -283,7 +296,9 @@ def test_send_alert_event_with_groupevent(self, safe_urlopen):
             f"/api/0/issues/{group_event.group.id}/"
         )
         assert data["data"]["event"]["issue_id"] == str(group_event.group.id)
-
+        assert data["data"]["event"]["occurrence"] == convert_dict_key_case(
+            group_event.occurrence.to_dict(), snake_to_camel_case
+        )
```

Comment:
Are we adding occurence to the webhook?

Reply:
yes we're now serializing occurrence here: https://github.com/getsentry/sentry/pull/80263/files#diff-b5fc1664529df00a2660afe503420f76540827b1292aa857a147c2229122ef04R78-R108